pkg/chaosdaemon/iptables_server.go

```diff
@@ -51,36 +53,51 @@ func (s *Server) FlushIptables(ctx context.Context, req *pb.IpTablesRequest) (*e
 		return nil, fmt.Errorf("unknown rule direction")
 	}
 
-	action := ""
 	switch rule.Action {
 	case pb.Rule_ADD:
-		action = "-A"
+		err = s.addIptablesRules(ctx, nsPath, format, rule)
```

Comment:
IMO, it's not a good idea to pass format and rule into addIptablesRules and build the command at addIptablesRules, i prefer to build the command before executing addIptablesRules.